pyproject.toml

```diff
@@ -43,7 +43,14 @@ dependencies = [
     "scipy>=1.10.1",
     "setuptools-scm>=7.1.0",
     "torch>=1.13.1",
-    "baybe[telemetry]",
+    # Following packages are necessary for telemetry. Due to issue with poetry
```

Comment:
these comments just explain the history of changes, not something anyone needs to know so I suggest removing

Reply:
Id see that ore in line witha nnote similar to the old audit-note a la
```
opentelemetry==a.b.c, # for opentelemetry
```

Reply:
Will change accordingly.

Reply:
I have written a very short TELEMETRY NOTE and added `# Telemetry` in the corresponding lines. Please resolve if this is fine.

Reply:
Shortened it even further 👍🏼